ci-operator/jobs/openshift/builder/openshift-builder-master-presubmits.yaml

```diff
@@ -6,15 +6,15 @@ presubmits:
     - ^master$
     - ^master-
     cluster: build03
-    context: ci/prow/e2e-aws
+    context: ci/prow/e2e-aws-builds
```

Comment:
was this change intentional?

Reply:
yeah, this part is ugly. make jobs re-arranges things with the new names. makes the diffs here in GH hard to read.